dev/scripts/src/alluxio.org/build-distribution/cmd/common.go

```diff
@@ -67,21 +79,24 @@ var ufsModules = map[string]module{
 	"ufs-hadoop-ozone-1.2.1": {"hadoop-ozone-1.2.1", "ozone", true, "-pl underfs/ozone -Pufs-hadoop-3 -Dufs.ozone.version=1.2.1"},
 }
 
+var coreLibJars = map[string]struct{}{
```

Comment:
how did you determine to separate these jars from the original list? seems like some but not all ufs jars are moved

Reply:
This is from the initial discussion with @beinan and @apc999 about what UFS to include first and give those UFSes for users to try out and for benchmarking. This PR is merged to fuse-sdk branch first and the UFSes are not the final decision.  For final product delivery, still need further discussion of what UFSes to include and possibly minimize to S3/Azure/GCS only.